torchvision/datasets/lsun.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, db_path, classes='train',
                 classes = [classes]
             else:
                 classes = [c + '_' + classes for c in categories]
-        if type(classes) == list:
+        elif type(classes) == list:
```

Comment:
hmm, should it be: `elif isinstance(classes, list): ..`, it more pythonic :)

Reply:
I was following the principle of minimal change :) but you're right, it's more pythonic that way. Anyway, it seems there is another bug down the road, maybe I'll wait to figure that out for the next commit.

Reply:
@iacolippo: Yup, i think so too :+1: